src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketRequestExecutor.java

```diff
@@ -149,7 +149,7 @@ public void makeDeleteRequest(HttpUrl url) {
 
     private void ensureNonEmptyBody(Response response) {
         if (response.body() == null) {
-            log.severe("Bitbucket - No content in response");
+            log.fine("Bitbucket - No content in response");
```

Comment:
Hmm, maybe `info` is better? I'm thinking of the support person having to answer "why didn't do what I thought it would do?" type questions. It is unlikely you run at `debug` level, and these should be rare so I think maybe `info` is better?